docs/pages/setup/reference/cli.mdx

```diff
@@ -685,6 +685,54 @@ registered devices using [`tsh mfa ls`](cli.mdx#tsh-mfa-ls).
 $ tsh mfa rm <device-name>
 ```
 
+### tsh exec
+
+Run provided command, by default using a shell, with Teleport-specific environment
+variables set. Can be used to create combined commands.
+
+The command can be used to run external executables, defaulting to a shell. To
+find a usable shell, `tsh exec` will try in order: `$SHELL` env variable, `sh`
+, `bash`. The flag `--shell` allows for overriding of the executable to run,
```

Comment:
To have `tsh` execute the command without a shell, would you set `--shell` to an empty string?

Reply:
`tsh exec` is for executing outside programs. By default, those programs are shells (`$SHELL`, `bash`, `sh`), but you can override the program to run using `--shell` flag. If you set `--shell` to empty string it willf fail to run anything because there is no "" executable.

The flag `--shell` is a bit of a misnomer, but I couldn't find a better name so I left it as is. I'd appreciate suggestions though.